<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>troubleshoot</class>
 <widget class="QWidget" name="troubleshoot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
General Stylesheet for Windows

To preview in designer: Copy/Paste this into the MainWindow widget stylesheet
To build: Mainwindow.cpp will copy the contents of this .qss into the MainWindow widget during initialization.
          Make sure to copy/paste changes from designer back to this stylesheet.
*/

QLabel {
        font-family: &quot;Open Sans&quot;;
        font-size: 12px;
        color: white;
        background-color: rgba(0,0,0,0);
}

QLabel#QuNexus_Not_Connected
{
        font: 500 16px &quot;Arial&quot;;
        color: white;
        border: none;
}

/* --[H1]------------------------- */
QLabel#Library_Pane_Label, QLabel#Send_Pane_Label,
QLabel#SEQ_settings_label, QLabel#SEQ_track_settings_label
 {
    background-color: rgba(0,0,0,0);
    border: none;
    color: qlineargradient(spread: pan, x1:0, y1:0, x2:1, y2:1,
                            stop:1 rgba(200,200, 200, 255),
                            stop:0 rgba(255,255,255, 255));
    font: italic 20px &quot;Arial&quot;;
}

/* --[H2]------------------------- */

QLabel#MIDI_To_CV_Label, QLabel#CV_To_MIDI_Label{
    background-color: rgba(0,0,0,0);
    border-top-left-radius:4.0;
    border-top-right-radius:4.0;
    border-bottom-left-radius:0;
    border-bottom-right-radius:0;
    border: none;
    color: rgb(225,225,225);
    font: italic 14px &quot;Arial&quot;;
}




/* --[PRESET LIBRARY] ------------- */

QListView#presetLibraryList {
        background: rgba(50, 50, 50, 255);
        min-height: 90px;
        margin: 0 0 0 0;
    border: 1px solid black;
        border-bottom-left-radius:4.0;
        border-bottom-right-radius:4.0;
        border-top-left-radius:4.0;
        border-top-right-radius:4.0;
        font-family: &quot;Arial&quot;;
        font-size: 14px;
}

QListView#presetLibraryList::item:hover {
    background:rgba(67, 67, 67, 255);
    color: rgba(175,175,175,255);
}

QListView#presetLibraryList::item{
        color: rgb(177, 177, 177);
        border: 0px;
}

QListView#presetLibraryList::item:selected, QListView:item:focus
 {
 background: rgba(162, 0, 0, 200);
 color: white;
 }

/*---------------------------------*/


QCheckBox
{
    background-color: transparent;
}

QToolTip {
        border: none;
    background: rgba(50,50,50,255);
    color: white;
}

QFrame QToolTip {
        border: none;
    background: rgba(50,50,50,255);
    color: white;
}

QPushButton {
        font-family: &quot;Arial&quot;;
        font-size: 8pt;
        background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(67, 67, 67, 255));
        border-style: outset;
        border-radius: 4.0;
        border-width: 1pt;
        border-color: black;
        color: white;
}

QPushButton:pressed {
        background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(67, 67, 67, 255), stop:1 rgba(150, 150, 150, 255));
}



/*--------[SCROLL BAR]---------------------------*/


QScrollBar:vertical {
        background: rgb(44,44,44);
/* margin sets up how far the handle can travel */
        margin: 0px 0px 0px 0px;
        border: none;
}

/* Sets up the color and height of handle */
QScrollBar::handle:vertical {
        background: rgb(115,115,115);
        border: none;
}

/* This remvoes the bottom button by setting the height to 0px */
QScrollBar::add-line:vertical {
        height: 0px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
        border: none;
}

/* This remvoes the top button by setting the height to 0px */
QScrollBar::sub-line:vertical {
        height: 0px;
        subcontrol-position: top;
        subcontrol-origin: margin;
        border: none;
}

/* need this to get rid of crosshatching on scrollbar background */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
        background: none;
        border: none;
}


/* -- [TABS] ------------------------------------- */

QTabWidget::pane {
        border-bottom-left-radius:4px;
        border-bottom-right-radius:4px;
        border-top-right-radius:4px;
}

QTabWidget::tab-bar {
        left: 0px;
        border: 0px;
}


QTabBar::tab {
        background: #262626;
        border-color: rgb(167, 255, 67);
        border-width:0px;
        border-top-left-radius: 4px;
        border-top-right-radius: 4px;
        width: 90px;
        height: 20px;
        padding: 4px;
        color: white;
        text-transform: uppercase;
        font: 500 11px &quot;Open Sans&quot;;


        margin-right: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
        background-color:rgba(120, 120, 120, 255);
        color:white;
}

QTabBar::tab:selected {
        background: rgba(67, 67, 67, 255);
        color:white;
}
</string>
  </property>
  <widget class="QFrame" name="troubleFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#QFrame#troubleFrame
{
	background-color: rgb(50,50,50);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="portsWindow">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>40</y>
      <width>308</width>
      <height>394</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Note - QTextEdit fonts are styled in  HTML */
    background-color: rgba(0,0,0,255);
    color: white;
border: black;

</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Source Code Pro'; font-size:8px;&quot;&gt;Initialized&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>10</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;</string>
    </property>
    <property name="text">
     <string>MIDI Ports</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;</string>
    </property>
    <property name="text">
     <string>Status Log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="statusWindow">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>40</y>
      <width>308</width>
      <height>394</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Note - QTextEdit fonts are styled in  HTML */
    background-color: rgba(0,0,0,255);
    color: white;
border: black;

</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Source Code Pro'; font-size:8px;&quot;&gt;Initialized&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>674</x>
      <y>10</y>
      <width>200</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;</string>
    </property>
    <property name="text">
     <string>Troubleshooting</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="troubleWindow">
    <property name="geometry">
     <rect>
      <x>674</x>
      <y>40</y>
      <width>308</width>
      <height>394</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Note - QTextEdit fonts are styled in  HTML */
    background-color: rgba(0,0,0,255);
    color: white;
	border: black;

</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Source Code Pro'; font-size:8px;&quot;&gt;Plug in device&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_close">
    <property name="geometry">
     <rect>
      <x>894</x>
      <y>443</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_gather">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>331</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: 12px &quot;Arial&quot;;
	color:white;
	border: none;
	background: rgba(0,0,0,0);
	text-align: left;
}
QPushButton:pressed
{
	color: red;
}</string>
    </property>
    <property name="text">
     <string>Click here to copy the diagnostic report to the Clipboard</string>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>portsWindow</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>statusWindow</zorder>
   <zorder>label_4</zorder>
   <zorder>troubleWindow</zorder>
   <zorder>butt_close</zorder>
   <zorder>butt_gather</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
